packages/block-library/src/post-excerpt/edit.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * WordPress dependencies
+ */
+import { useEntityProp, useEntityId } from '@wordpress/core-data';
+import { RichText } from '@wordpress/block-editor';
+
+function PostExcerptDisplay() {
+	const [ excerpt ] = useEntityProp( 'postType', 'post', 'excerpt' );
+	return <RichText.Content tagName="p" value={ excerpt } />;
+}
+
+export default function PostExcerptEdit() {
+	if ( ! useEntityId( 'postType', 'post' ) ) {
+		return 'Post Excerpt Placeholder';
```

Comment:
i18n

Reply:
I don't think we should use it here.

This is a temporary placeholder while #19256 is worked on.